### PR TITLE
chores: improve setup

### PR DESCRIPTION
**Pull request by divyenduz**

### CODE REVIEW

File: .github/scripts/test-all.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo "$packages" | tr ' ' '\n' | while read -r item; do
   echo "running $item"
 
   case "$item" in
-    *".github"*|*"experimental"*|*"deployment-platforms"*|"typescript/subscriptions-pubsub")
+    *".github"*|*"experimental"*|*"deployment-platforms"*|*"typescript/subscriptions-pubsub"*|*"typescript/postgis-express"*|*"typescript/graphql"*|*"typescript/graphql-nextjs"*|*"javascript/graphql"*|*"javascript/graphql-apollo-server"*|*"javascript/graphql-auth"*)
```
Comment (janpio) [inline .github/scripts/test-all.sh:20]: What is the reasoning behind ignoring all these?

File: .github/tests/javascript/graphql-auth/run.sh [hunk 2]
```diff
@@ -9,4 +9,4 @@ sleep 15
 
 curl --fail 'http://localhost:4000/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:4000' --data-binary '{"query":"query {\n  feed {\n    id\n    content\n    author {\n      id\n      name\n      email\n    }\n  }\n}"}' --compressed
 
-kill "$pid"
+kill "$pid" || true
```
Comment (janpio) [inline .github/tests/javascript/graphql-auth/run.sh:12]: Why should this ever not work?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: testing
+    volumes:
+      - postgres:/var/lib/postgresql/data
+volumes:
+  postgres:
```
Comment (janpio) [inline docker-compose.yml:15]: Why is this needed?

File: typescript/graphql-apollo-server/schema.graphql [hunk 4]
```diff
@@ -3,10 +3,8 @@
 
 
 type Mutation {
-  createDraft(authorEmail: String, content: String, title: String!): Post!
-  deleteOnePost(where: PostWhereUniqueInput!): Post
+  createDraft(authorEmail: String!, content: String, title: String!): Post!
   publish(id: Int): Post
-  signupUser(data: UserCreateInput!): User!
```
Comment (janpio) [inline typescript/graphql-apollo-server/schema.graphql:9]: What are all these deletions in this file?

File: .github/scripts/test-all.sh (hunk 1 above)
Comment (divyenduz) [reply .github/scripts/test-all.sh:20]: `*".github"*|*"experimental"*|*"deployment-platforms"*` I don't know, that is how it was when I picked it. 

Rest use `nexus-prisma` which is broken at the moment.

File: .github/tests/javascript/graphql-auth/run.sh (hunk 2 above)
Comment (divyenduz) [reply .github/tests/javascript/graphql-auth/run.sh:12]: No idea but it failed on my local once and the existing code has a catch-all to clean up. This is just a precautionary measure.

File: docker-compose.yml (hunk 3 above)
Comment (divyenduz) [reply docker-compose.yml:15]: For easier local setup. The Pg in the tests comes from actions service but when I want to set it up locally I need to translate that to a `docker-compose.yml` file.

File: typescript/graphql-apollo-server/schema.graphql (hunk 4 above)
Comment (divyenduz) [reply typescript/graphql-apollo-server/schema.graphql:9]: I didn't make the change, it is most likely a "generated" change

File: .github/scripts/test-all.sh (hunk 1 above)
Comment (nikolasburk) [reply .github/scripts/test-all.sh:20]: Actually, we can keep those after all since we "fixed" the corresponding tests by reverting the versions of the `@prisma` packages back to `2.7.0`. Let's rather keep them in the tests and resolve the issues by reverting to a lower version rather than removing tests. (Sorry I think I wasn't clear about this before).

Comment (divyenduz) [reply .github/scripts/test-all.sh:20]: Noted, will do so in a new PR.

File: .github/tests/javascript/graphql-auth/run.sh (hunk 2 above)
Comment (janpio) [reply .github/tests/javascript/graphql-auth/run.sh:12]: Please add an issue to investigate and understand this (or replace that code).

File: docker-compose.yml (hunk 3 above)
Comment (janpio) [reply docker-compose.yml:15]: Root of the repository is not an appropriate location for this then as it gives a wrong impression about what these examples need or want to be run.

File: typescript/graphql-apollo-server/schema.graphql (hunk 4 above)
Comment (janpio) [reply typescript/graphql-apollo-server/schema.graphql:9]: This should be understood then before committing - it changes the example and might be wanted - or not.

Comment (divyenduz) [reply typescript/graphql-apollo-server/schema.graphql:9]: `"generate:nexus": "ts-node --transpile-only src/schema"` this is in the `package.json` file, I think this is generated. I call @nikolasburk to validate.

File: docker-compose.yml (hunk 3 above)
Comment (divyenduz) [reply docker-compose.yml:15]: Removed in https://github.com/prisma/prisma-examples/pull/2068

File: .github/tests/javascript/graphql-auth/run.sh (hunk 2 above)
Comment (divyenduz) [reply .github/tests/javascript/graphql-auth/run.sh:12]: Removed in https://github.com/prisma/prisma-examples/pull/2068

File: .github/scripts/test-all.sh (hunk 1 above)
Comment (divyenduz) [reply .github/scripts/test-all.sh:20]: Removed all the nexus ones via https://github.com/prisma/prisma-examples/pull/2079

The "postgis" one is most likely something else (I need to investigate). Will remove that in a separate PR.
